src/lib.rs

```diff
@@ -21,12 +20,8 @@ pub fn run(opts: Opts) -> Result<()> {
             SubCommand::Activate { name } => match name {
                 Some(name) => commands::activate(&name)?,
                 None => {
```

Comment:
We can probably make a bit of a stylistic change here now, like:

```rust
let name = name.unwrap_or(skim.fuzzy_find()?);
commands::activate(&name);
```

(On mobile, not tested, but that's the gist)